### PR TITLE
remove qtauto image from available images / reformat README

### PR DESCRIPTION
**Pull request by tobsan**
With b2qt being conditional in meta-pelux, we only show this image as
suggested image when we are building with b2qt support. This requires a
change in the manifest to copy that file.

Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,21 @@ Maintained at https://github.com/pelagicore/meta-pelux
...
 Copyright (C) 2016 Pelagicore AB
 
-All metadata is MIT licensed unless otherwise stated. Source code included in tree for individual recipes is under the LICENSE stated in the associated recipe (.bb file) unless otherwise stated.
+All metadata is MIT licensed unless otherwise stated. Source code included in
+tree for individual recipes is under the LICENSE stated in the associated
+recipe (.bb file) unless otherwise stated.
 
-License information for any other files is either explicitly stated or defaults to MIT license.
+License information for any other files is either explicitly stated or defaults
+to MIT license.
```
Comment (GordanM) [inline README.md]: Formatting changes should be a separate commit.

**Review by GordanM — CHANGES_REQUESTED** [review]

**Comment by tobsan** [comment]
@GordanM separated the commits now.

**Review by GordanM — APPROVED** [review]
